src/components/Tasks/Tasks.js

```diff
@@ -0,0 +1,67 @@
+import { useContext, useState } from 'react';
+import { Redirect } from 'react-router-dom';
+import { UserContext } from '../../context/UserContext';
+import { createTodo, fetchTodos, toggleTasks } from '../../services/client';
+import { useTodos } from '../../hooks/useTodos';
+import './Tasks.css';
+
+export default function Tasks() {
+  const [todo, setTodo] = useState('');
+  const [complete, setComplete] = useState(false);
```

Comment:
I can't find any place you're using this `complete` state except for one line below where you `setComplete(false)`, but it's already false. Do you need to have a separate piece of state to track this? (consider all the objects in your `todos` a property of `complete` that is already keeping track of whether they are completed or not)